test/e2e/util.go

```diff
@@ -174,3 +190,41 @@ func getDeployment(t *testing.T, client client.Client, name types.NamespacedName
 	}
 	return &dep, nil
 }
+
+// Send HTTPS Request to a route and check what is returned in the Response
+func getHttpResponse(t *testing.T, route *routev1.Route) (http.Header, int, string, error) {
+	t.Helper()
+	if len(route.Spec.Host) == 0 {
+		return nil, 0, "", fmt.Errorf("route.Spec.Host is empty, cannot test route")
+	}
+	request, err := http.NewRequest("GET", "https://"+route.Spec.Host, nil)
+	if err != nil {
+		return nil, 0, "", fmt.Errorf("error creating HTTPS request %v: %v", request, err)
+	}
+
+	// Send the HTTPS request
+	timeout, _ := time.ParseDuration("10s")
+	client := &http.Client{
+		Timeout: timeout,
```

Comment:
This can be: `Timeout: 10*time.Second`. No need to parse as above.

Reply:
done